cpm_generic/templates/cpm_generic/tags/mainmenu.html

```diff
@@ -8,8 +8,15 @@
   {% endfor %}
 </ul>
 <ul class="nav pull-right">
-  <li class="active">
-    <a href="/">Home</a>
-  </li>
+  {% for page, is_active in menupages %}
+  <li>{% if is_active %}<strong>{% endif %}
```

Comment:
Because there is no style for "active" nav item (#32) so there is no visual feedback for currently selected menu item.